packages/frontend/editor-ui/src/components/Node/NodeCreator/viewsData.ts

```diff
@@ -639,7 +639,7 @@ export function RegularView(nodes: SimplifiedNodeType[]) {
 		type: 'view',
 		properties: {
 			title: i18n.baseText('nodeCreator.triggerHelperPanel.addAnotherTrigger'),
-			icon: 'bolt',
+			icon: 'bolt-filled',
```

Comment:
types here did not capture this. Too much work to fix. Created a follow up tech debt story